contrib/bootstrap.fedora.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# Originated from https://git.io/autobuild 
+#
+# This script performs rpmbuild environment setup and the initial autotools bootstrapping.
+# Abort the script and exit with failure if any command below exits with
+# a non-zero exit status.
+set -e
+OS=`cat /etc/redhat-release`
```

Comment:
I think it's better to assign this variable only after checking if the file exists in line 10